lib/sequent/test/command_handler_helpers.rb

```diff
@@ -125,9 +125,12 @@ def when_command command
         @repository.clear
       end
 
-      def then_events *events
-        expect(@event_store.stored_events.map(&:class)).to eq(events.flatten(1).map(&:class))
-        @event_store.stored_events.zip(events.flatten(1)).each do |actual, expected|
+      def then_events(*expected_events)
+        expected_classes = expected_events.flatten(1).map { |event| event.class == Class ? event : event.class }
```

Comment:
@derekkraan normally I would suspect `A === a` and `a === A` to do the same thing, but then again it might be that `A` provides its own `===` implementation.

Maybe go with `Class === event` instead of `event.class == Class`?